examples/src/main/kotlin/org/dashevo/examples/GetStatus.kt

```diff
@@ -6,42 +6,66 @@
  */
 package org.dashevo.examples
 
+import com.google.common.base.Stopwatch
+import org.bitcoinj.params.DevNetParams
 import org.dashevo.Client
+import org.dashevo.dapiclient.model.GetStatusResponse
+import org.dashevo.dapiclient.provider.DAPIAddress
 
+/*
+    Calls getStatus on every masternode in the default list for a devnet
+
+    Reports time to reply in ms for each node
+ */
 class GetStatus {
     companion object {
-        val sdk = Client("palinka")
+        lateinit var sdk: Client
 
         @JvmStatic
         fun main(args: Array<String>) {
+            if (args.isEmpty()) {
+                println("Usage: GetStatus network")
+                return
+            }
+            sdk = Client(args[0])
             getStatus()
         }
 
-        fun getStatus() {
+        private fun getStatus() {
             sdk.isReady()
+            val results = hashMapOf<DAPIAddress, GetStatusResponse?>()
 
-            val total = 20
+            val nodeList = (sdk.platform.params as DevNetParams).defaultMasternodeList.toMutableList()
+            nodeList.add("211.30.243.82")
```

Comment:
Was this line for testing purpose?

Reply:
Yes, it will be removed.  Good catch!

Reply:
Fixed here: https://github.com/dashevo/android-dashpay/pull/29/commits/a7b2b8285b6af1ad07d3cb08f5b9bd7d26190b0f